lib/output.js

```diff
@@ -1185,6 +1200,21 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        var parent = output.parent(1);
```

Comment:
So if `n` is out of range then `undefined` would be returned?

You should check for that case in DEFPRINT(AST_String)